src/Spec-MorphicAdapters/MorphicTextInputFieldAdapter.class.st

```diff
@@ -29,8 +29,7 @@ MorphicTextInputFieldAdapter >> adapt: aModel [
 { #category : #factory }
 MorphicTextInputFieldAdapter >> buildWidget [
 	| plu |
-	plu := PluggableTextFieldMorph new
-		convertTo: String;
+	plu := RubPluggableTextFieldMorph new
```

Comment:
The old PluggableTextFieldMorph implemented `maxLength:` and Rub's does not...

Reply:
The same with `hideScrollBarsIndefinitely`

Reply:
It also does not have a getter for `ghostText` (which we require for testing that we set the right thing)

Reply:
We also are using `widget setTextAndAccept: aString`

Reply:
Then... there is another more fundamental problem with this change, and it is the fact that it may kill Spec's compatibility with Pharo7.

Hmm well, looking at it, we have
```smalltalk 
canDiscardEdits
	^self hasUnacceptedEdits not
```
So if we can add the missing methods as extension methods in a compatibility package, that may be enough.